.github/workflows/bump-version.yml

```diff
@@ -16,3 +16,24 @@ jobs:
           sed: 's/^\(const version = "\).*\("\)$/\1$VERSION\2/g'
           file: app_info.go
           author_email: bender.github@secrethub.io
+  release-aws:
+    name: Release zip file for lambda deployment
+    runs-on: ubuntu-latest
+    needs: bump-version
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Make zip for AWS-Lambda
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.13'
+        run: go biuld -o lambda-webhook cmd/lambda
+        run: zip secrethub-kubernetes-mutating-webhook-${{ env.RELEASE_VERSION }}-lambda.zip lambda-webhook
+      - name: Upload Release Asset
+        uses: actions/upload-release-asset@v1
+        with:
+        asset_path: ./secrethub-kubernetes-mutating-webhook-${{ env.RELEASE_VERSION }}-lambda.zip
+        asset_name: secrethub-kubernetes-mutating-webhook-${{ env.RELEASE_VERSION }}-lambda.zip
+        asset_content_type: application/zip
```

Comment:
This shouldn't be in the `bump-version` workflow, but placed in a new workflow, e.g. `release.yml`.

Reply:
Adjusted.